src/app/learn/reference/configuration/index.html

```diff
@@ -257,6 +254,34 @@
           </stache-code-block>
         </li>
       </ul>
+      <p>
```

Comment:
This LGTM John.  What do you think about supplementing it with something like https://github.com/blackbaud/skyux2-docs/pull/130?

Reply:
Added this content to the parameters section